docs/schema.rst

```diff
@@ -0,0 +1,20 @@
+.. _schema:
+
+===============
+Schema Creation
+===============
+
+Here we create connection to SQLite database client and then we discover & initialize models.
+
+.. automethod:: tortoise.Tortoise.generate_schemas
+    :noindex:
+
+``generate_schema`` generates schema on empty database, you shouldn't run it on every app init, run it just once, maybe out of your main code.
```

Comment:
is there any major downside to running it regardless? i currently run it all the time to not have to write an install script for new instances but just have tortoise figure it out during startup

Reply:
Er, you don'g get a big error if your db lost its contents?

Reply:
this only is for new instances, like if someone starts it up for the first time then it'll just create the database on startup instead of crashing cause it doesn't exist and requiring them to run a seperate script

just wondering since you mention here you shouldn't do this if there is a downside to this design pattern